### PR TITLE
Added quality assurance and testing questions

### PR DESCRIPTION
**Pull request by jayantpranjal0**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

### CODE REVIEW

File: src/data/quality-assurance-quiz.ts [hunk 1]
```diff
@@ -116,7 +116,16 @@ const qualityAssuranceQuiz = [
     Explanation:
       "Test-driven development is the act of first deciding what you want your program to do (the specifications), formulating a failing test, then writing the code to make that test pass.",
     Link: "https://www.freecodecamp.org/news/an-introduction-to-test-driven-development-c4de6dce5c/"
-  }
+  },
+  {
+    Question: "Which of the following is true about TDD?",
+    Answer:"With TDD, test logic precedes application logic.",
+    Distractor1:"With TDD application logic precedes test logic",
```
Comment (jdwilkin4) [inline src/data/quality-assurance-quiz.ts:123]:
```suggestion
    Distractor1:"With TDD, application logic precedes test logic",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Comment by jayantpranjal0** [comment]
Thank you for your recommendation. I have changed the file as per your recommendation. Please review/merge the PR.

**Review by jdwilkin4 — APPROVED** [review]
Congrats on another contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**